src/main/java/core/basesyntax/AreaCalculator.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface AreaCalculator {
+    double getArea();
```

Comment:
Better name `calculateArea` since you make calculations, not just getting the filed value from object.

Reply:
![image](https://user-images.githubusercontent.com/69212372/218399098-32748d20-24fe-4a21-b3f0-67f9c543eeee.png)

so?